### PR TITLE
Replaced curl with more robust wget

### LINKED ISSUE #1252: Dependencies Download Issues on Mac on Travis
**Describe the Bug**
In the mac test on Travis it often happens that one of the sumo, xc16 or ros dependencies fail to download, see for example: https://travis-ci.com/cyberbotics/webots/jobs/274046592#L3816

### PR DESCRIPTION
**Pull request by omichel**
Attempt to fix #1252.

### CODE REVIEW

**Comment by omichel** [comment]
👍 ✔️

File: projects/robots/gctronic/e-puck/transfer/Makefile [hunk 1]
```diff
@@ -19,30 +19,22 @@ include $(WEBOTS_HOME_PATH)/resources/Makefile.os.include
 
 XC16_TARGET=xc16
 XC16_VERSION=1.24
+XC16_PACKAGE=$(XC16_TARGET)-$(XC16_VERSION).zip
+PREVIOUS_XC16_PACKAGE=$(filter-out $(XC16_PACKAGE),$(wildcard XC16-*.zip))
+EXTRACT=unzip -q
+WGET=wget --quiet
 
 ifeq ($(OSTYPE),linux)
  HOSTTYPE ?= $(shell uname -m)
```
Comment (DavidMansolino) [inline projects/robots/gctronic/e-puck/transfer/Makefile]:
```suggestion
```

File: projects/default/resources/Makefile [hunk 2]
```diff
@@ -19,43 +19,35 @@ include $(WEBOTS_HOME_PATH)/resources/Makefile.os.include
...
+WGET=wget --quiet
 
 ifeq ($(OSTYPE),linux)
  HOSTTYPE ?= $(shell uname -m)
  OS_TAG=linux64
  SUMO_PACKAGE=sumo-$(SUMO_VERSION)-linux64.tar.gz
  PREVIOUS_SUMO_PACKAGE=$(filter-out $(SUMO_PACKAGE),$(wildcard sumo-*-linux64.tar.gz))
  EXTRACT=tar xfz
- WGET=wget --quiet
+ SUMO_SOURCE=$(basename $(basename $(SUMO_PACKAGE)))
```
Comment (DavidMansolino) [inline projects/default/resources/Makefile]:
```suggestion
 # remove .tar.gz
 SUMO_SOURCE=$(basename $(basename $(SUMO_PACKAGE)))
```

File: projects/default/resources/Makefile [hunk 3]
```diff
@@ -19,43 +19,35 @@ include $(WEBOTS_HOME_PATH)/resources/Makefile.os.include
...
 endif
 
 ifeq ($(OSTYPE),darwin)
  OS_TAG=mac
  SUMO_PACKAGE=sumo-$(SUMO_VERSION)-mac.tar.gz
  PREVIOUS_SUMO_PACKAGE=$(filter-out $(SUMO_PACKAGE),$(wildcard sumo-*-mac.tar.gz))
  EXTRACT=tar xfz
- WGET=curl -O --silent
+ SUMO_SOURCE=$(basename $(basename $(SUMO_PACKAGE)))
```
Comment (DavidMansolino) [inline projects/default/resources/Makefile]:
```suggestion
 # remove .tar.gz
 SUMO_SOURCE=$(basename $(basename $(SUMO_PACKAGE)))
```

File: projects/default/resources/Makefile [hunk 4]
```diff
@@ -19,43 +19,35 @@ include $(WEBOTS_HOME_PATH)/resources/Makefile.os.include
...
  OS_TAG=windows
  SUMO_PACKAGE=sumo-$(SUMO_VERSION)-windows.zip
  PREVIOUS_SUMO_PACKAGE=$(filter-out $(SUMO_PACKAGE),$(wildcard sumo-*-windows.zip))
  EXTRACT=unzip -q
- WGET=wget --quiet
-endif
-
-ifeq ($(OSTYPE),windows)
- # remove .zip
  SUMO_SOURCE=$(basename $(SUMO_PACKAGE))
```
Comment (DavidMansolino) [inline projects/default/resources/Makefile]:
```suggestion
 # remove .zip
 SUMO_SOURCE=$(basename $(SUMO_PACKAGE))
```

**Comment by omichel** [comment]
👍

**Pull request merged**